lib/Sema/DerivedConformanceCodable.cpp

```diff
@@ -0,0 +1,950 @@
+#include "TypeChecker.h"
```

Comment:
What do you mean, like `#include "swift/.../TypeChecker.h"`? This is not a public header; it is in `lib/Sema/TypeChecker.h`, right alongside `lib/Sema/DerivedConformanceCodable.cpp`. This is how all other clients using `TypeChecker` import it.